### PR TITLE
dev: add mock framework rpc + block number test

### PR DESCRIPTION
**Pull request by dubzn**
# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [x] Other (please describe): Wiremock was added to mock RPC responses
# What is the current behavior?

Issue Number: [#243](https://github.com/keep-starknet-strange/beerus/issues/243)

# What is the new behavior?
In the new behavior, mocks can be added to the wiremock file just [like in Kakarot-RPC ](https://github.com/sayajin-labs/kakarot-rpc/blob/50a3bf8fb74c3836e1e10042de44c6c10744c123/core/src/utils/wiremock_utils.rs#L111)(the path to the .json with the response must be added).

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: beerus-rpc/tests/utils.rs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+pub async fn setup_beerus_rpc() -> BeerusRpc {
+    let mock_starknet_rpc = setup_wiremock().await;
+    let config = Config {
+        ethereum_network: "goerli".to_string(),
+        ethereum_consensus_rpc: "".to_string(),
+        ethereum_execution_rpc: "".to_string(),
+        starknet_rpc: mock_starknet_rpc,
+        starknet_core_contract_address: Default::default(),
+        data_dir: Some(PathBuf::from(DEFAULT_DATA_DIR)),
+    };
```
Comment (dubzn) [inline beerus-rpc/tests/utils.rs:22]: It needs to be validated, the config has several attributes that I didn't know how to fill, so they were left blank.

Comment (ftupas) [reply beerus-rpc/tests/utils.rs:22]: Maybe the `new_from_env` method might be helpful to instantiate a mutable `Config` object and then you just pass in the mock starknet rpc

Comment (dubzn) [reply beerus-rpc/tests/utils.rs]: Thanks @ftupas! I made a small refactor and I liked more how it looks now, but feedback is always welcome :smiley:

**Comment by dubzn** [comment]
I close this PR and open a new one with the same changes but updated with the code refactor! :P https://github.com/keep-starknet-strange/beerus/pull/265

**Pull request closed without merging**